dynamiq/nodes/agents/orchestrators/linear_manager.py

```diff
@@ -199,6 +312,14 @@ def __init__(self, **kwargs):
         """
         super().__init__(**kwargs)
         self._init_prompt_blocks()
+        self._init_actions()
```

Comment:
Parent class call `self._init_actions()` on __init__, so you shouldn't call it again